playerstate.lua

```diff
@@ -0,0 +1,37 @@
+
+
+
+
+
+g_PlayerStates = {}
```

Comment:
Make this `local` - it should not be directly accessed from the outside of this file.